gp-easy-passthrough/gpep-edit-entry.php

```diff
@@ -40,6 +40,9 @@ public function __construct( $options ) {
 
 		add_filter( "gpi_query_{$this->form_id}", array( $this, 'exclude_edit_entry_from_inventory' ), 10, 2 );
 
+		// If we need to reprocess any feeds on 'edit'.
+		add_filter( 'gform_is_feed_asynchronous', array( $this, 'check_feed_async' ), 10, 2 );
```

Comment:
If this snippet is loaded it's going to disable async feeds everywhere. Remember to keep these types of filters properly scoped as we do in https://github.com/gravitywiz/snippet-library/blob/master/gp-entry-blocks/gpeb-process-feeds-on-edit.php